python/lsst/drp/tasks/single_frame_detect_and_measure.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, schema=None, **kwargs):
             type="U",
             doc="Detector this source appeared on.",
         )
-
+        afwTable.CoordKey.addErrorFields(self.schema)
```

Comment:
I think you meant `afwTable.CoordKey.addErrorFields(schema)`?

Reply:
Good catch! Thank you.